### DISCUSSION TITLE
Recommended way of customizing links to source file?

### DISCUSSION THREAD

**Discussion by matthew-brett** [executablebooks/meta#106] [Q&A] [answered]
I want to be able to customize what the students see with the link to download the notebook source files.

In my class, I ask the students to download notebooks and run them on their laptops.

To make their lives easier, during the build step, I ran a post-processing script on the source notebooks, to replace relative links to files with their full URL equivalents.  For example, a link to a data file would change from the equivalent of `<../data/my_file.csv>` to `<https://matthew-brett.github.io/cfd2019/data/my_file.csv>`.

But - this requires me to customize the Download Source link from the actual source, to the preprocessed source, with a different path.  What is the recommended way to customize that link?

**Comment by chrisjsewell** [comment]
I believe this is a duplicate of this issue: https://github.com/executablebooks/jupyter-book/issues/886?

**Comment by matthew-brett** [comment, marked as answer]
Yes it is - sorry for not spotting that one.
